erpnext/accounts/doctype/bank_reconciliation_tool/bank_reconciliation_tool.js

```diff
@@ -155,7 +155,7 @@ frappe.ui.form.on("Bank Reconciliation Tool", {
 		}
 	},
 
-	render_chart: frappe.utils.debounce((frm) => {
+	render_chart(frm) {
```

Comment:
This is a hack because render_chart is called twice on load. That is a bug that needs to be fixed rather than a hack to hide the problem.
It's not an animation.